frontend/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
```

Comment:
Are all of these unmodified django files necessary if this is going to be all typescript?

Reply:
I am quite sure that I will have to add some frontend-only models like "User announcements" at some point so I would like to keep them for now.